packages/core/test/editable-text/editableTextTests.tsx

```diff
@@ -75,7 +75,7 @@ describe("<EditableText>", () => {
             const OLD_VALUE = "alphabet";
             const NEW_VALUE = "hello";
 
-            const component = mount(
+            const component = mount<EditableText>(
```

Comment:
Oof, this is annoying. Hope you get a response to your question in DefinitelyTyped.

Reply:
I think it's unlikely to be resolved and we'll have to keep doing this